core/parser.lua

```diff
@@ -6020,6 +6020,17 @@ local SPELL_POWER_PAIN = SPELL_POWER_PAIN or (PowerEnum and PowerEnum.Pain) or 1
 		Details222.MythicPlus.BackgroundTexture = backgroundTexture
 
 		if (time) then
+            --Subtract death time from time of run to get the true time
+            local deaths = C_ChallengeMode.GetDeathCount()

```

Comment:
Yes this function gives a timeLost value. It is not accurate for 7+ dungeons with the new affix increasing death time lost.